### PR TITLE
infoschema: Add empty table REFERENTIAL_CONSTRAINTS

### PR DESCRIPTION
**Pull request by shenli**
Add a empty table REFERENTIAL_CONSTRAINTS in information_schema database to make some java connector happy.
Add test case to check infomation_schema tables.

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
@zimulala PTAL

File: infoschema/infoschema_test.go [hunk 1]
```diff
@@ -224,6 +224,43 @@ func (testSuite) TestConcurrent(c *C) {
...
+		"PARTITIONS",
+		"KEY_COLUMN_USAGE",
+		"REFERENTIAL_CONSTRAINTS",
+	}
+	for _, t := range info_tables {
+		tb, err1 := is.TableByName(model.NewCIStr(infoschema.Name), model.NewCIStr(t))
+		c.Assert(err1, IsNil)
+		c.Assert(tb, NotNil)
+	}
+
```
Comment (zimulala) [inline infoschema/infoschema_test.go]: Remove the empty line.

**Comment by shenli** [comment]
@zimulala PTAL

**Comment by hanfei1991** [comment]
LGTM

**Pull request merged**